Content.Server/WoodBurner/ActiveWoodBurnerComponent.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Content.Server.WoodBurner
+{
+    [RegisterComponent]
+    public sealed class ActiveWoodBurnerComponent : Component
+    {
+    }
```

Comment:
Instead of having a separate component for `ActiveWoodBurnerComponent`, consider having a simple `isActive` bool on your main Component. Something similar is done in `EvaporationComponent` with `EvaporationToggle`.

Reply:
Will it allow to skip this component in Update event without pressure on performance?

Reply:
> Instead of having a separate component for `ActiveWoodBurnerComponent`, consider having a simple `isActive` bool on your main Component. Something similar is done in `EvaporationComponent` with `EvaporationToggle`.

This is not the convention we wish to follow. Use the active component.